form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/web/rest/FormFlowResource.kt

```diff
@@ -57,7 +57,7 @@ class FormFlowResource(
         return ResponseEntity.ok(GetFormFlowStateResult(instance.id.id, openStep(stepInstance)))
     }
 
-    @PostMapping("/v1/form-flow/{formFlowId}/step/{stepInstanceId}")
+    @PostMapping(value = ["/v1/form-flow/instance/{formFlowId}/step/instance/{stepInstanceId}","/v1/form-flow/{formFlowId}/step/{stepInstanceId}"]) // TODO: deprecate "/v1/form-flow/{formFlowId}/step/{stepInstanceId}"
```

Comment:
So there's still some TODOs in here, can you do those? I think they're a part of this story right?